app/config/services/services.xml

```diff
@@ -141,6 +141,7 @@
         <service id="AppBundle\Command\ReferentTagInitializeCommitteeEvent"/>
         <service id="AppBundle\Referent\ReferentTagManager" />
         <service id="AppBundle\Referent\ManagedCommitteesExporter" public="true" />
+        <service id="AppBundle\Referent\ManagedEventsExporter" public="true" />
```

Comment:
normally, you don't need to use public here, 
you can redeclare the same service with prefix `test.AppBundle\Referent\ManagedEventsExporter` in `config_test.yml` with `public: true` config.